### PR TITLE
Re-Structure NLog also added ExceptionLoggerFilter 

### LINKED ISSUE #165: NLog re-stucturing
All logs right now are written directly in Error log. Implement a proper folder structure as per oauth server and different logfile for different level.

### PR DESCRIPTION
**Pull request by ankit213**
Fixed #165

Changed Files : 

1.**ExceptionLoggerFilter.cs** : Added ExceptionLoggerFilter file for global exception
2.**NLog.config**: Updated NLog Confgi File.
3.**Global.asax.cs** Updated Application_Start Method(added exception logger filter in globalFilter.filter)

### CODE REVIEW

**Review by chintans — APPROVED** [review]

File: Slack.Automation/Promact.Erp.Core/ActionFilters/ExceptionLoggerFilter.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using Autofac.Extras.NLog;
+using System.Web.Mvc;
+
+namespace Promact.Erp.Core.ActionFilters
+{
+    public class ExceptionLoggerFilter : IExceptionFilter
+    {
+        private ILogger _iLogger;
```
Comment (chintans) [inline Slack.Automation/Promact.Erp.Core/ActionFilters/ExceptionLoggerFilter.cs]: Why it is not readonly?

Comment (chintans) [reply Slack.Automation/Promact.Erp.Core/ActionFilters/ExceptionLoggerFilter.cs]: Name should have been _logger

File: Slack.Automation/Promact.Erp.Web/App_Start/AutofacConfig.cs [hunk 2]
```diff
@@ -90,9 +90,9 @@ public static IComponentContext RegisterDependancies()
             builder.RegisterModule<NLogModule>();
             builder.RegisterModule<SimpleNLogModule>();
             builder.Register(x => AutoMapperConfiguration.ConfigureMap()).As<IMapper>().SingleInstance();
-
+            //builder.RegisterFilterProvider();
```
Comment (chintans) [inline Slack.Automation/Promact.Erp.Web/App_Start/AutofacConfig.cs]: remove commented code

**Review by chintans — CHANGES_REQUESTED** [review]

File: Slack.Automation/Promact.Erp.Core/ActionFilters/ExceptionLoggerFilter.cs (hunk 1 above)
Comment (ankit213) [reply Slack.Automation/Promact.Erp.Core/ActionFilters/ExceptionLoggerFilter.cs]: It should be readonly ,
It's my mistake, I have updated

File: Slack.Automation/Promact.Erp.Web/App_Start/AutofacConfig.cs (hunk 2 above)
Comment (ankit213) [reply Slack.Automation/Promact.Erp.Web/App_Start/AutofacConfig.cs]: I have removed this comment

**Review by chintans — APPROVED** [review]

**Pull request merged**